src/edu/csus/ecs/pc2/core/report/JudgementReport.java

```diff
@@ -67,6 +66,12 @@ public void writeReport(PrintWriter printWriter) {
             printWriter.print(hiddenText+" id=" + judgement.getElementId());
             printWriter.println();
         }
+        
+        printWriter.println();
+        printWriter.println("# reject.ini");
+        printWriter.println("#");
+        printRejectINI(printWriter, contest);
```

Comment:
I think it's a bit of a stretch to say that addition of a whole new method which alters how a report appears falls under the category of the declared issue (#119), which is
`There are non-logged exceptions.
There are compiler warnings in the project.
`

Reply:
I did not make this change.